From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 7 Jun 2019 09:51:33 +0200
Subject: fix build after y2038 changes in glibc
Patch-mainline: upstream seems to be dead

SIOCGSTAMP is now defined in linux/sockios.h. Include that file wherever
needed.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 src/maint/atmdump.c  |    1 +
 src/maint/saaldump.c |    1 +
 2 files changed, 2 insertions(+)

--- a/src/maint/atmdump.c
+++ b/src/maint/atmdump.c
@@ -16,6 +16,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h> /* for htonl and ntohl */
 #include <atm.h>
+#include <linux/sockios.h>
 
 
 static const char *pti[] = { "Data SDU 0","Data SDU 1","Data SDU 0, CE",
--- a/src/maint/saaldump.c
+++ b/src/maint/saaldump.c
@@ -16,6 +16,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <atm.h>
+#include <linux/sockios.h>
 
 #include "pdu.h"
 #define DUMP_MODE
